go/logger/standard.go

```diff
@@ -355,37 +355,37 @@ func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
 }
 
 func (log *UnforwardedLogger) Debug(s string, args ...interface{}) {
-	if log.internal != nil {
+	if log != nil && log.internal != nil {
```

Comment:
is the original problem that `log` is `nil` and not that `log.internal` is `nil`?